www/main/components/addons/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as SundarGutka } from './sundar-gutka';
```

Comment:
what are "addons" ?

Reply:
Addons are just a name used for "Sunder Gutka", "ceremonies", "sync" etc. I see them as things that add functionality to the core STTM functionality (search and view).

Can use a different name, open to suggestions.